ui/components/MesherySettings.js

```diff
@@ -137,6 +138,11 @@ class MesherySettings extends React.Component {
               <FontAwesomeIcon icon={faPoll} transform={mainIconScale} fixedWidth />
             } label="Metrics" />
           </Tooltip>
+          <Tooltip title="Performance" placement="top">
+            <Tab icon={
+              <FontAwesomeIcon icon={faPoll} transform={mainIconScale} fixedWidth />
```

Comment:
Here's an icon that we've created for this tab - https://github.com/layer5io/meshery/blob/master/ui/static/img/performance.svg

Reply:
this is done!

Reply:
Though i don't know how to solve the above issue. I am facing the same one for a while now

Reply:
@Nikhil-Ladha suggestions here?

Reply:
Actually, I haven't yet tested it on my local. So, once i have done that I will share my feedback.